anta/tests/routing/generic.py

```diff
@@ -187,3 +188,90 @@ def test(self) -> None:
             self.result.is_success()
         else:
             self.result.is_failure(f"The following route(s) are missing from the routing table of VRF {self.inputs.vrf}: {missing_routes}")
+
+
+class VerifyRouteEntry(AntaTest):
+    """Verifies the route entries of given IPv4 network(s).
+
+    Supports `strict: True` to verify that only the specified nexthops by which routes are learned, requiring an exact match.
+
+    Expected Results
+    ----------------
+    * Success: The test will pass if the route entry with given nexthop(s) present for given network(s).
+    * Failure: The test will fail if the routes not found or route entry with given nexthop(s) not present for given network(s).
+
+    Examples
+    --------
+    ```yaml
+    anta.tests.routing:
+      generic:
+        - VerifyRouteEntry:
+            route_entries:
+                - prefix: 10.10.0.1/32
+                  vrf: default
+                  nexthops:
+                    - 10.100.0.8
+                    - 10.100.0.10
+    ```
+    """
+
+    name = "VerifyRouteEntry"
+    description = "Verifies the route entry(s) for the provided IPv4 Network(s)."
+    categories: ClassVar[list[str]] = ["routing"]
+    commands: ClassVar[list[AntaCommand | AntaTemplate]] = [AntaTemplate(template="show ip route {prefix}", revision=4)]
```

Comment:
we can give the vrf information into command so that will again help to reduce the output if a network is configure with another vrf